### ISSUE TITLE
Minified React error #418 and double loaded css file

### ISSUE THREAD

**Issue by kmergen** [remix-run/remix#4262]
### What version of Remix are you using?

1.7.1

### Steps to Reproduce

I try to rewrite my app from nextjs to remix. I get the o. a. error in Chrome console. I also see in the network tab that my global css file is double loaded.

To reproduce please see your example application https://remix-ecommerce.fly.dev/en/search.
![Screenshot 2022-09-22 161050](https://user-images.githubusercontent.com/1640101/191770260-f5e8f647-2110-400d-915e-66f7f770cb26.jpg)

### Expected Behavior

no console error and no double loaded css

### Actual Behavior

see screenshot.

**Comment by kogan007** [comment]
This is happening to me as of right now too, and it didn't before. Those errors show up in my console on the ecommerce demo as well.

**Comment by vishnu-acodez** [comment]
@kogan007 Did you find the soution?

**Comment by sylvainDNS** [comment]
I had this error and fixed it with the eslint rule: `'@typescript-eslint/no-duplicate-imports': 'error'`.
In two files, I had duplicated import like this : 
```ts
import { Icon } from '@chakra-ui/react'
import { Link } from '@chakra-ui/react'
```

Now there is no longer error in the build  🥳

**Comment by WesleyYue** [comment]
I'm seeing this as well, and this only happens on production (when the files are minimized). My dev environment runs fine. Did anyone here find the issue?

**Comment by mateosantosdev** [comment]
Same error here on production

**Comment by huyb1991** [comment]
I got the same issue on Production

**Comment by WillSmithTE** [comment]
Likewise seemingly randomly happening on production

**Comment by jdnichollsc** [comment]
Hey folks, we're getting the same error from our DEV environment:
```
chunk-OELJV5DJ.js:8 Uncaught Error: Minified React error #423; visit https://reactjs.org/docs/error-decoder.html?invariant=423 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at $a (chunk-OELJV5DJ.js:8:44889)
    at Ha (chunk-OELJV5DJ.js:8:39459)
    at Qf (chunk-OELJV5DJ.js:8:39390)
    at el (chunk-OELJV5DJ.js:8:39250)
    at Ai (chunk-OELJV5DJ.js:8:35673)
    at Va (chunk-OELJV5DJ.js:8:34629)
    at _l (chunk-OELJV5DJ.js:1:1743)
    at MessagePort.gl (chunk-OELJV5DJ.js:1:2133)
```
Can you reopen this issue please? <3

**Comment by raufabr** [comment]
I'm having the same issue at the moment - I'm using Remix with Hydrogen and certain browser extensions, such as Lightdash are causing the app to crash.
![Uploading Screenshot 2023-07-18 at 11.43.12.png…]()

**Comment by kocyigityunus** [comment]
in our case, it was happening because we were trying to push some current time info to the dashboard. which explains why it's random. sometimes server rendered and hydrated data matches, sometimes don't.

**Comment by sylvainDNS** [comment]
I finally found the problem: like @raufabr it was a browser extension that broke the first render (Dark Reader extension, that inject css to provide dark theme)

**Comment by agoyal118** [comment]
How can I see the non-minified error?

**Comment by kiliman** [comment]
Click on the link in the dev console to see the full error message.

**Comment by dkkazmierczak** [comment]
I guess as well as @sylvainDNS it can be a browser extension, when you go to incognito the issue disappear.

**Comment by abstain23** [comment]
I got same error, but when you go to incognito the issue disappear..
![image](https://github.com/user-attachments/assets/eeb3f61d-6008-4c84-a8df-8846bff2c1e0)

**Comment by muhajirdev** [comment]
Did you fix the issue @abstain23 ?

**Comment by nigellima** [comment]
Getting the same issue here, only in production. And the error happens in any browser I've tried, including in Incognito.

Any ideas?

**Comment by valendion** [comment]
Same here

**Comment by b-mendoza** [comment]
Disabling all my browser extensions and using incognito mode doesn't work for me. **The hydration error persists.**

I've noticed that what my error logs have in common is that they originate from a `MessagePort` call and come from the `entry.client` file.

I found that it seems the error is being caused by the React Scheduler, specifically a utility called `unstable_now`.

**Comment by TheBuilderJR** [comment]
I'm running into the same issue w/ https://telemetry.sh

loom here: https://t.co/isoBFeutft

**Comment by TheBuilderJR** [comment]
Can we at least not crash the entire site when this happens? Feels like a re-render is ok.

**Comment by bran-watcha** [comment]
In my case, the issue was caused by the `gtag` script added for analytics. 

The gtag script inserts an `iframe tag` into the document. 

It seems that this action occurs before hydration rather than after, causing the issue. 

This happens because the scope of hydration currently covers the entire `document`. 

We postponed the gtag initialize script until after hydration, and the issue no longer occurs. 

Extensions that modify the document might also cause minified errors.

I hope this helps 🙏
